### PR TITLE
Adds a testcase for HTTP multiple content-length

### PR DESCRIPTION
**Pull request by catenacyber**
This is not right yet : there remains `to_server` to be added in the rule.

It seems that the alert mixes source and destination of the packet :
Resulting eve.json contains :
```
  "event_type": "alert",
  "src_ip": "127.0.0.1",
  "src_port": 8000,
  "dest_ip": "127.0.0.1",
  "dest_port": 50333,
```
 but packet 11 has 50333 as source port and 8000 as destination

Making Suricata dump the packet gives
RQAAKAAAAABABnzOfwAAAX8AAAEfQMSdU3Yied5LJohQEAoASTEAAA==

The hexdump is
 ```
0000000 45 00 00 28 00 00 00 00 40 06 7c ce 7f 00 00 01
0000010 7f 00 00 01 1f 40 c4 9d 53 76 22 79 de 4b 26 88
0000020 50 10 0a 00 49 31 00 00                        
```
So it looks like Suricata rewrote the packet switching its fields from 
```
0000   02 00 00 00 45 00 01 33 00 00 40 00 40 06 00 00
0010   7f 00 00 01 7f 00 00 01 c4 9d 1f 40 de 4b 26 88
0020   53 76 22 79 80 18 31 d5 ff 27 00 00 01 01 08 0a
```

### CODE REVIEW

File: tests/http-multiple-cl/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Description
+
+Test HTTP parsing against multiple different content-length.
+
+# PCAP
+
+The pcap comes from running `python -m SimpleHTTPServer 8000` as a server and the present dummy python script client.py
```
[the remark this answers is not in the document]
Comment (catenacyber) [reply tests/http-multiple-cl/README.md]: Ok. Any better idea to get a simple (yet supporting HTTP1.1) HTTP server running ?

File: tests/http-multiple-cl/client.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+print "3", data
+#repetition of another header
+#a = sock.send("POST /which.php HTTP/1.1\r\nHost: 127.0.0.1\r\nContent-Lol:%d\r\nContent-Lol:%d\r\nContent-Length:%d\r\n\r\n%s\r\n" % (len(dummyContent), len(dummyContent), len(dummyContent), dummyContent))
+#simple repetition
+#a = sock.send("POST /which.php HTTP/1.1\r\nHost: 127.0.0.1\r\nContent-Length:%d\r\nContent-Length:%d\r\n\r\n%s" % (len(dummyContent), len(dummyContent), dummyContent))
+a = sock.send("GET /ok2.php HTTP/1.1\r\nUser-Agent: Mozilla\r\nHost: 127.0.0.1\r\n\r\n")
+data = sock.recv(2000)
+print "4", data
+
+sock.close()
```
[the remark this answers is not in the document]
Comment (catenacyber) [reply tests/http-multiple-cl/client.py]: Ok. Was python a bad idea here ?

[the remark this answers is not in the document]
Comment (victorjulien) [reply tests/http-multiple-cl/client.py]: No I think we can be less strict here than with code we actually use. This is more for reference.

[the remark this answers is not in the document]
Comment (catenacyber) [reply tests/http-multiple-cl/client.py]: This was indeed intended as a quick and dirty documentation to the pcap

**Pull request closed without merging**